[BUG] Installer fails: ERROR: Failed building wheel for evdev

## 🐛 Bug Description
The installer fails and gives an error.

## 📋 Steps to Reproduce
1. In terminal, run the command `curl -fsSL raw.githubusercontent.com/jatinkrmalik/vocalinux/main/install.sh -o /tmp/vl.sh && bash /tmp/vl.sh --tag=v0.5.0-beta`

## ✅ Expected Behavior
Installer should complete successfully.

## ❌ Actual Behavior
Installer fails partway through the install process with the following output:
```
Building wheels for collected packages: vocalinux, evdev, pyaudio, srt
  Building wheel for vocalinux (pyproject.toml) ... done
  Created wheel for vocalinux: filename=vocalinux-0.5.0b0-py3-none-any.whl size=463106 sha256=d584842320dc256239d56ecf8c53f77a1630fec591a5b0b35d2898093abc2745
  Stored in directory: /home/jordan/.cache/pip/wheels/12/30/d2/70bfe30d7f2ead426961fd896a4494444ce9a16ddf40d77b07
  error: subprocess-exited-with-error

  × Building wheel for evdev (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> No available output.

  note: This error originates from a subprocess, and is likely not a problem with pip.
  Building wheel for evdev (pyproject.toml) ... error
  ERROR: Failed building wheel for evdev
  Building wheel for pyaudio (pyproject.toml) ... done
  Created wheel for pyaudio: filename=pyaudio-0.2.14-cp312-cp312-linux_x86_64.whl size=69521 sha256=74cf5a0e5d81252010df93c48d4fd1d9bb8957d2fa97edec68db789986c5b181
  Stored in directory: /home/jordan/.cache/pip/wheels/68/c7/33/c6a6b210cb5819ec5c219928c794a447742a7d86d21c0b92e6
  Building wheel for srt (pyproject.toml) ... done
  Created wheel for srt: filename=srt-3.5.3-py3-none-any.whl size=22512 sha256=b063b9c8b995c9bb9dac6fea01f51ecbf588fbdb54f0aa16d10996b7295e0a62
  Stored in directory: /home/jordan/.cache/pip/wheels/7e/75/5b/e1d5c3756631e4bda806f6cc9640153b39484bb6f7b0b8def3
Successfully built vocalinux pyaudio srt
Failed to build evdev
error: failed-wheel-build-for-install

× Failed to build installable wheels for some pyproject.toml based projects
╰─> evdev
[ERROR] Failed to install Vocalinux.
[ERROR] Check the pip log for details: /tmp/tmp.9n0H50XroY/pip_log.txt
[ERROR] Failed to install Vocalinux package. Installation cannot continue.
```

## 📸 Screenshots
N/A

## 💻 Environment
- **OS**: PopOS 24.04
- **Python Version**: 3.12.3
- **Vocalinux Version**: whichever version is installed by the install script, probably 0.5.0-beta
- **Display Server**: Wayland
- **Speech Engine**: N/A
- **Model Size**: N/A

## 📋 Debug Logs
See installer error above

## 📝 Additional Context
N/A

--- a/install.sh
+++ b/install.sh
@@ -524,12 +524,12 @@ install_system_dependencies() {
     fi
 
     # Define package names for different distributions
-    local APT_PACKAGES_UBUNTU="python3-pip python3-gi python3-gi-cairo gir1.2-gtk-3.0 gir1.2-appindicator3-0.1 libgirepository1.0-dev python3-dev portaudio19-dev python3-venv pkg-config wget curl unzip"
-    local APT_PACKAGES_DEBIAN_BASE="python3-pip python3-gi python3-gi-cairo gir1.2-gtk-3.0 libcairo2-dev python3-dev portaudio19-dev python3-venv pkg-config wget curl unzip"
+    local APT_PACKAGES_UBUNTU="python3-pip python3-gi python3-gi-cairo gir1.2-gtk-3.0 gir1.2-appindicator3-0.1 libgirepository1.0-dev python3-dev build-essential portaudio19-dev python3-venv pkg-config wget curl unzip"
+    local APT_PACKAGES_DEBIAN_BASE="python3-pip python3-gi python3-gi-cairo gir1.2-gtk-3.0 libcairo2-dev python3-dev build-essential portaudio19-dev python3-venv pkg-config wget curl unzip"
     local APT_PACKAGES_DEBIAN_11_12="$APT_PACKAGES_DEBIAN_BASE libgirepository1.0-dev gir1.2-ayatanaappindicator3-0.1"
     local APT_PACKAGES_DEBIAN_13_PLUS="$APT_PACKAGES_DEBIAN_BASE libgirepository-2.0-dev gir1.2-ayatanaappindicator3-0.1"
     local DNF_PACKAGES="python3-pip python3-gobject gtk3 libappindicator-gtk3 gobject-introspection-devel python3-devel portaudio-devel python3-virtualenv pkg-config wget curl unzip"
-    local PACMAN_PACKAGES="python-pip python-gobject gtk3 libappindicator-gtk3 gobject-introspection python-cairo portaudio python-virtualenv pkg-config wget curl unzip"
+    local PACMAN_PACKAGES="python-pip python-gobject gtk3 libappindicator-gtk3 gobject-introspection python-cairo portaudio python-virtualenv pkg-config wget curl unzip base-devel"
     local ZYPPER_PACKAGES="python3-pip python3-gobject python3-gobject-cairo gtk3 libappindicator-gtk3 gobject-introspection-devel python3-devel portaudio-devel python3-virtualenv pkg-config wget curl unzip"
     # Gentoo uses Portage and different package naming convention
     local EMERGE_PACKAGES="dev-python/pygobject:3 x11-libs/gtk+:3 dev-libs/libappindicator:3 media-libs/portaudio dev-lang/python:3.8 pkgconf"

